jsparty/js-party-81.md

```diff
@@ -246,6 +246,6 @@ Have you ever gone into a Node modules folder and tried to fix something? Becaus
 
 **Jerod Santo:** How do people reach you online?
 
-**George Mandis:** People can google me, because a long time ago I think I got basically every George Mandis username and domain that's out there, at some point... But if you go to george.mand.is - that's my site and blog. Anything I'm working on, you can find there. You can also find me on Twitter, where I don't say a lot, but I'm on there.
+**George Mandis:** People can google me, because a long time ago I think I got basically every George Mandis username and domain that's out there, at some point... But if you go to [george.mand.is](george.mand.is) - that's my site and blog. Anything I'm working on, you can find there. You can also find me on Twitter, where I don't say a lot, but I'm on there.
```

Comment:
Looks like this became a relative link to the transcripts repo, suggested fix:
```suggestion
**George Mandis:** People can google me, because a long time ago I think I got basically every George Mandis username and domain that's out there, at some point... But if you go to [george.mand.is](https://george.mand.is) - that's my site and blog. Anything I'm working on, you can find there. You can also find me on Twitter, where I don't say a lot, but I'm on there.
```

Reply:
Whelp, I didn't realize I could commit changes from PR. That would've been simpler.

I rebased from `thechangelog/master` then committed and pushed the suggested fix, which was probably not the best way to go about it.

Reply:
No worries, end result looks right!